### ISSUE TITLE
Missing value?

### ISSUE THREAD

**Issue by arachnophilia1997** [sicreative/VoltageShift#50]
Hi. Looking at the manual.

This _sudo ./voltageshift buildlaunchd  -50 -50 0 0 0 0 50 80 160_ appears to be missing one value. It should be 10 values, should it not?

**Comment by thisbansal** [comment]
Yes. It needs ten arguments. You have only nine in there.
And btw, does cpu undervolting works for you?

**Comment by arachnophilia1997** [comment]
Yes it works! I'm on a dell laptop running macOS catalina. It works after doing some of this bios modding: https://www.reddit.com/r/Dell/comments/fzv599/xps_7590_160_uefi_unlock_undervolting_and_remove/
